.github/workflows/_build-production-node-and-runtime.yml

```diff
@@ -27,7 +27,10 @@ jobs:
           targets: wasm32-unknown-unknown
 
       - name: Build production binary and runtime
-        run: cargo build --profile production -p aleph-node
+        # yamllint disable-line rule:line-length
+        run: |
+          cargo build --profile production -p aleph-node
+          nm --demangle=rust target/release/aleph-node | grep -q liminal && { echo "Liminal symbols are present"; exit 1; }
```

Comment:
I think this should be `target/production/aleph-node` instead `target/release/aleph-node`

Reply:
oh, you're right, thank you for spotting; fixed